pkg/asset/internal/templates.go

```diff
@@ -539,12 +540,16 @@ spec:
     spec:
       containers:
       - name: etcd-operator
-        image: quay.io/coreos/etcd-operator:c391d8b7638deb81aa877773a0acce389f602415
+        image: quay.io/coreos/etcd-operator:dev
```

Comment:
This has some offline discussion as context.

First of all, latest beta release (the only beta release) has some client-go conflict.

We don't have nightly job to push to this tag yet, so we manually test latest etcd operator, build and push this tag.

We will change this to v0.2.1 once it's released, and keep this tag unchanged before that.